test/unit/annotation_spec.js

```diff
@@ -90,16 +91,19 @@ describe('annotation', function() {
         startObjId: 0,
       });
 
-      var annotation1 = AnnotationFactory.create(xref, annotationDict,
-                                                 pdfManagerMock, idFactory);
-      var annotation2 = AnnotationFactory.create(xref, annotationDict,
-                                                 pdfManagerMock, idFactory);
-      var data1 = annotation1.data, data2 = annotation2.data;
-      expect(data1.annotationType).toEqual(AnnotationType.LINK);
-      expect(data2.annotationType).toEqual(AnnotationType.LINK);
+      AnnotationFactory.create(xref, annotationDict,
+                pdfManagerMock, idFactory).then((annotation) => {
```

Comment:
Indent this like you did here: https://github.com/mozilla/pdf.js/pull/9417/files#diff-04ad2b4c022ab1eae0d58931b8fe0b7dR74